src/resolve/test-dns-packet-extract.c

```diff
@@ -2092,4 +2092,720 @@ TEST(packet_reply_rrsig_signer_overflow) {
         ASSERT_EQ(dns_answer_size(packet->answer), 0u);
 }
 
+/* ================================================================
+ * reply: SVCB/HTTPS
+ * ================================================================ */
+
+static DnsSvcParam* add_svcb_param(DnsResourceRecord *rr, uint16_t key, const char *value, size_t len) {
+        DnsSvcParam *param = calloc(1, offsetof(DnsSvcParam, value) + len);
```

Comment:
calloc/malloc/realloc need to have their result checked too

Reply:
(again because this is a test, it's fine to assert here rather than returning an error